HaulageApplication/Haulage/DatabaseExecutionServices/ExpenseExecutionService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SQLite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Haulage.BaseClasses;
+using Haulage.BaseClasses.Accounting;
+using Haulage.BaseClasses.BillingHandler;
+
+namespace Haulage.DatabaseExecutionServices
+{
+    public static class ExpenseExecutionService
+    {
+        public static List<Expense> GetExpenses()
+        {
+            var expenses = new List<Expense>();
+            var sql = "SELECT [ExpenseId], [DriverId], [VehicleId], [Fullname], [Expense], [ExpenseDescription], [ExpenseCost] FROM [Expense];";
+
+            using (var connection = new SQLiteConnection(DatabaseSetup.GetDatabasePath()))
+            {
+                var command = new SQLite.SQLiteCommand(connection);
+                command.CommandText = sql;
+                expenses = command.ExecuteQuery<Expense>();
+            }
+            return expenses;
+        }
+        public static void DeleteExpense(int expenseId)
+        {
+            var sql = $"DELETE FROM [Expense] WHERE [ExpenseId] = {expenseId};";
+
+            using (var connection = new SQLiteConnection(DatabaseSetup.GetDatabasePath()))
+            {
+                var command = new SQLite.SQLiteCommand(connection);
+                command.CommandText = sql;
+                command.ExecuteNonQuery();
+            }
+
+        }
+        public static void SaveExpense(Expense expense)
```

Comment:
the ability for the user to edit an expense could be useful if an error was made when entering in data. This could be future work

Reply:
will add as a new development task